src/main/java/uk/gov/ons/ctp/response/casesvc/representation/action/NotifyModel.java

```diff
@@ -115,8 +115,8 @@ public Classifiers build() {
             return new Classifiers(actionType, "", "", "", "");
           }
 
-          // if not not england is england. welcome to ras-rm.
-          if (!NOT_ENGLAND.equals(region)) {
+          // if NOT NI then England
```

Comment:
I'd put England/Wales just to be 100% clear

Reply:
okay - ideally we wouldnt need the comment but its already been confusing

Reply:
Agreed